doc/examples/applications/plot_cornea_spot_inpainting.py

```diff
@@ -0,0 +1,258 @@
+"""
+============================================
+Restore spotted cornea image with inpainting
+============================================
+
+Optical coherence tomography (OCT) is a non-invasive imaging technique used by
+ophthalmologists to take pictures of the back of a patient's eye [1]_.
+When performing OCT,
+dust may stick to the reference mirror of the equipment, causing dark spots to
```

Comment:
We have referred to the obstruction as "dirt" everywhere else in the tutorial.
```suggestion
dirt may stick to the reference mirror of the equipment, causing dark spots to
```

Reply:
Hmm, no. This is the *introductory* section, so this is where we lay out the context and define/introduce the recurring terms. Therefore, in the *following* sections, we'll have to be consistent with this introduction (which sets the tone for the rest). In the next sentence, 'these' (in "these dirt spots") make it clear that we are referring to the (just mentioned) dark spots caused by dust. And we'll stick with "dirt spots" in the rest of the tutorial.

Reply:
PS: By "no" I mean "not necessarily." It wouldn't be *wrong* to replace "dust" by "dirt," it's just not necessary (without breaking overall consistency).